src/Realm.C

```diff
@@ -63,6 +63,7 @@
 
 // actuator line/fsi
 #include <aero/AeroContainer.h>
+#include <aero/fsi/OpenfastFSI.h>
```

Comment:
```suggestion
```
This header should not be needed.  All dependencies on FSI code should be isolated to the AeroContainer